Pickers/AbstractActionSheetPicker.m

```diff
@@ -218,7 +254,7 @@ - (void)showActionSheetPicker {
 
 #pragma clang diagnostic push
 #pragma ide diagnostic ignored "UnavailableInDeploymentTarget"
-    if ([UIViewController instancesRespondToSelector:@selector(edgesForExtendedLayout)]) {
```

Comment:
This line was previously added by me for a work-around of tap dismiss bug (by disabling tap dismiss action completely on older devices). But after the commit 7c7d892 (by foxluqi), it was no longer needed because that commit resolved this issue properly on all devices